cmd/extensions/main.go

```diff
@@ -170,6 +172,23 @@ func main() {
 		health = healthcheck.NewHandler()
 	}
 
+	podReady = true
+	health.AddReadinessCheck("agones-extensions", func() error {
+		if !podReady {
+			return errors.New("asked to shut down, failed readiness check")
+		}
+		return nil
+	})
+
+	signals.NewSigTermHandler(func() {
+		logger.Info("pod shutdown has been requested, failing readiness check")
```

Comment:
total nit, but s/pod/Pod/ for consistency with other log messages.

Reply:
Done!